client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/show-hide-input/style.scss

```diff
@@ -0,0 +1,58 @@
+.show-hide-input {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	outline: solid 1px var(--studio-gray-10);
+	padding: 8px;
+	border-radius: 4px;
+	gap: 10px;
+	height: 28px;
+	max-width: 456px;
+	transition: all 0.2s;
+
+	&--hidden {
```

Comment:
Nit: I have a strong personal preference for avoiding `&` as selector/text duplicator in CSS selectors because it makes it much harder to find the selector when debugging in an IDE.

While that may not seem like a common use case, every now and then we hit issues with CSS selectivity in Calypso as a result of duplicate selector names that are defined in different CSS bundles, and knowing where and how the rules were defined is relatively important.

Also, I find I need to think extra-hard to make sure I know what `&` actually represents, especially within patch diffs where the wrapping selector may not be visible.

Reply:
Thanks for the context:
I removed https://github.com/Automattic/wp-calypso/pull/88952/commits/b24b270f9e140f52eb4ea1f4dc624f7a8f1e3b21 the usage of `&`